src/Pages/Dashboard.jsx

```diff
@@ -1,35 +1,68 @@
 import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import {  STOCKS_API, STOCKS_TOKEN, CALL_ME_SERVER} from "../utils/constants"
 
 export default function Dashboard() {
-
   const [stocks, setStocks] = useState([]);
+  const [quantityToTrack, setQuantityToTrack] = useState(null);
 
-  const STOCKS_API = process.env.REACT_APP_FINNHUB_API;
-  const STOCKS_TOKEN = process.env.REACT_APP_FINN_TOKEN;
+  const navigate = useNavigate();
 
   useEffect(() => {
     fetch(`${STOCKS_API}/stock/peers?symbol=AAPL&token=${STOCKS_TOKEN}`)
       .then((res) => res.json())
       .catch(console.error())
       .then((symbols) => {
-        
-        const cleanSymbols = symbols.filter( symbol => symbol !== '1337.HK');
-
-        console.log({cleanSymbols});
+        const cleanSymbols = symbols.filter((symbol) => symbol !== "1337.HK");
 
         const symbolFetches = cleanSymbols.map((symbol) => {
-            return fetch(`${STOCKS_API}/quote?symbol=${symbol}&token=${STOCKS_TOKEN}`).then((res) => res.json()).then( prices => ({symbol, prices}))
-            }) 
-
-    Promise.all(symbolFetches).then(data => {
-        console.log({data});
-        setStocks(data);
-      })
-    })
+          return fetch(
+            `${STOCKS_API}/quote?symbol=${symbol}&token=${STOCKS_TOKEN}`
+          )
+            .then((res) => res.json())
+            .then((prices) => ({ symbol, prices }));
+        });
 
+        Promise.all(symbolFetches).then((data) => {
```

Comment:
Nice use of `Promise.all` here

Reply:
<img src="https://media3.giphy.com/media/3osxYdXvsGw6wT5lIY/giphy.gif"/>